composer.json

```diff
@@ -6,7 +6,7 @@
     "license": "MIT",
     "platform-check": false,
     "require": {
-        "php": "^7.3|^8.0",
```

Comment:
please use php 7.4 in this case. As it was identified on one of the last lesson, your image for docker is php 7.4

Reply:
Ok